docs/user/security/audit-logging.asciidoc

```diff
@@ -248,6 +332,74 @@ Refer to the corresponding {es} logs for potential write errors.
 | `success` | User has accessed an alert as part of a search operation.
 | `failure` | User is not authorized to access alerts.
 
+.2+| `case_get`
+| `success` | User has accessed a case.
+| `failure` | User is not authorized to access a case.
+
+.2+| `case_resolve`
+| `success` | User has accessed a case.
+| `failure` | User is not authorized to access a case.
+
+.2+| `case_find`
+| `success` | User has accessed a case as part of a search operation.
+| `failure` | User is not authorized to search for cases.
+
+.2+| `case_ids_by_alert_id_get`
+| `success` | User has accessed cases as part of a search operation.
+| `failure` | User is not authorized to search for cases.
+
+.2+| `case_get_metrics`
```

Comment:
Wasn't sure if this one should just be classified as `accessing a case` or if it should be more like:

`User has accessed metrics for a case`. Technically the metrics are being computed for the case so the user has accessed the case.

Reply:
If it's important for the audit log to be as specific as possible, "User has accessed metrics for a case" seems more informative to me.